### ISSUE TITLE
Error when setting xy value on hue Lamp

### ISSUE THREAD

**Issue by asgothian** [Jey-Cee/ioBroker.deconz#20]
Changing the xy datapoint on a hue lamp causes a warning from the restAPI:

```
2019-01-08 17:17:56.716 - debug: deconz.0 stateChange deconz.0.Light_1.xy {"val":"0.245041,0.242574","ack":false,"ts":1546967876711,"q":0,"from":"system.adapter.admin.0","lc":1546967876711}
2019-01-08 17:17:56.717 - debug: deconz.0 dp: xy; id:Light_1
2019-01-08 17:17:56.726 - info: deconz.0 setLightState: {"transitiontime": 0, "xy": "0.245041,0.242574"} 1 deconz.0.Light_1.xy
2019-01-08 17:17:56.738 - debug: deconz.0 STATUS: 400
2019-01-08 17:17:56.739 - info: deconz.0 options: {"url":"http://localhost:80/api/78CC3AFC48/lights/1/state","method":"PUT","headers":"Content-Type\" : \"application/json","body":"{\"transitiontime\": 0, \"xy\": \"0.245041,0.242574\"}"}
2019-01-08 17:17:56.739 - debug: deconz.0 setLightState BODY: [{"error":{"address":"/lights/1/state/xy","description":"invalid value, 0.245041,0.242574, for parameter, xy","type":7}}]
2019-01-08 17:17:56.740 - warn: deconz.0 Set light state with ID: 1 parameter: {"transitiontime": 0, "xy": "0.245041,0.242574"} Code 400: Bad request
```

The target value was copied from the setting of the object, but the formatting in the message does not match up with the formatting in the response to getLightState:
```
debug: deconz.0 getLightState: {"ctmax":500,"ctmin":153,"etag":"e720da66da458021a7b387fa0e21c362","hascolor":true,"manufacturername":"Philips","modelid":"LCT015","name":"Hobby Room Color","state":{"alert":"none","bri":254,"colormode":"hs","ct":153,"effect":"none","hue":40655,"on":true,"reachable":true,"sat":168,"xy":[0.234363,0.289572]},"swversion":"1.46.13_r26312","type":"Extended color light","uniqueid":"00:17:88:01:03:12:4d:c3-0b"}
```

**Comment by Jey-Cee** [comment]
#https://github.com/Jey-Cee/ioBroker.deconz/commit/f57c746d5f3b7167a7b63458627d5c50d03e33eb

**Comment by asgothian** [comment]
I tested this, but still got the same error.

`2019-01-08 20:52:07.077 - warn: deconz.0 Set light state with ID: 1 parameter: {"xy": ["0.634999,0.264986"]} Code 400: Bad request`

I then went ahead and changed the lines in main.js locally from
`parameters = '{"xy": [' + JSON.stringify(state.val) + ']}';`
to 
`parameters = '{"xy": [' + state.val + ']}';`
and 
`parameters = '{"transitiontime": ' + JSON.stringify(ttime) + ', "xy": [' + JSON.stringify(state.val) + ']}';`
to
`parameters = '{"transitiontime": ' + JSON.stringify(ttime) + ', "xy": [' + state.val + ']}';`

This change makes it work with well formed values. I also tried entering all kinds of odd strings. The behavior was consistent and beneficial. Any entry which was not x,y with x and y being numbers between 0.0 and 1.0 resulted in an error like the two examples below
```
2019-01-08` 21:47:08.620 - warn: deconz.0 Set light state with ID: 1 parameter: {"xy": [\'hall\'o]} Code 400: Bad request
2019-01-08 21:45:02.658 - warn: deconz.0 {"address":"/lights/1","description":"invalid value, [100,100], for parameter, /lights/1/xy","type":7}
```
In all the other cases, the color of the lamp changed and the values were accepted. In case of the lamp not being able to reach the exact coordinate, the xy coordinates would later update / to what i assume to be the closest reachable coordinate.
